tests/test_resources.py

```diff
@@ -1752,13 +1752,17 @@ def test_plan(self):
             name='Mock Plan',
             setup_fee_in_cents=Money(0),
             unit_amount_in_cents=Money(1000),
-            total_billing_cycles=10
+            total_billing_cycles=10,
+            custom_fields=[CustomField(name='size', value='large')],
```

Comment:
for the internal tests it maybe doesnt matter so much, more important for the playground, but consider using `food` here.

Reply:
Good point. Updated to `food`.